gulpfile.js

```diff
@@ -1014,19 +1014,26 @@ gulp.task('lib', ['buildnumber'], function () {
   var preprocessor2 = require('./external/builder/preprocessor2.js');
 
   return merge([
+    gulp.src('external/streams/streams-lib.js', {base: '.'})
+      .pipe(gulp.dest('build/')),
     gulp.src([
       'src/{core,display}/*.js',
       'src/shared/{compatibility,util}.js',
       'src/{pdf,pdf.worker}.js',
-    ], {base: 'src/'}),
+    ], {base: 'src/'})
```

Comment:
too many irrelevant changes. let's make original marge as `var buildLib = merge([...]);` and then add
```
return merge([
   buildLib,
   gulp.src('external/streams/streams-lib.js', {base: '.'})
     .pipe(gulp.dest('build/'))
  ]);
```